compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -418,8 +426,21 @@ static void build_accessors(AggregateType* ct, Symbol *field) {
     return;
 
   // Otherwise, build compiler-default getter and setter.
-  build_accessor(ct, field, true);
-  build_accessor(ct, field, false);
+
+  if (isUnion(ct)) {
+    // Unions need a special getter and setter.
+    build_accessor(ct, field, /* setter */ false, false);
+    build_accessor(ct, field, /* setter */ true,  false);
```

Comment:
Shouldn't one of these comments say "getter" instead?  If it is intended to label the boolean argument, why is there not also one for the second boolean, and an "="?